infra/upload_images/infra.py

```
@@ -473,6 +465,11 @@ def __init__(
                 "OPENROUTER_BASE_URL": "https://openrouter.ai/api/v1",
                 "OPENROUTER_MODEL": "openai/gpt-oss-120b",
                 "LANGCHAIN_API_KEY": langchain_api_key,
+                "LANGCHAIN_TRACING_V2": "true",  # Enable Langsmith tracing (LangChain)
+                "LANGSMITH_TRACING": "true",  # Enable Langsmith tracing (@traceable decorator)
+                "LANGCHAIN_PROJECT": label_validation_project_name or "receipt-validation",
+                "OPENROUTER_API_KEY": openrouter_api_key,
```

Ruff (0.14.13)

Dictionary key literal `"OPENROUTER_API_KEY"` repeated

Remove repeated key literal `"OPENROUTER_API_KEY"`

(F601)